examples/research_projects/onnxruntime/text_to_image/train_text_to_image.py

```diff
@@ -412,6 +412,8 @@ def main():
 
     if args.gradient_checkpointing:
         unet.enable_gradient_checkpointing()
+        vae.enable_gradient_checkpointing()
+        vae.requires_grad_(True)
```

Comment:
```suggestion
```

Think even without `requires_grad_` this should help as gradients are passed down

Reply:
@patrickvonplaten 
The original issue #1883 references being able to train the VAE. How can we incorporate training specifically the VAE into this script?

Subsequently, if `requires_grad_(False)` implies a frozen VAE (e.g. no backprop is done) and the purpose of gradient checkpointing is saving memory when computing the gradient of the loss by backprop doesn't that void the entire purpose of gradient checkpointing?

Sorry for the questions but this is my first PR so I'm a trying to understand the specifics.

Reply:
@patrickvonplaten @williamberman 
I think my above comment got lost in the resolved comment. Sorry about that.